### PR TITLE
Remove Resources dependency from OrchardCore.Email

### PR DESCRIPTION
**Pull request by hishamco**
Address https://github.com/OrchardCMS/OrchardCore/discussions/10437

### CODE REVIEW

**Comment by hishamco** [comment]
Good catch

**Comment by deanmarcussen** [comment]
Yeah no don’t merge this. You break the taghelpers by removing the reference in the csproj.

the dependency in the manifest can go but not the other changes

**Comment by Skrypt** [comment]
TagHelpers that might be used in the email templates? I think you are right.
So in this case there is no need.

**Comment by hishamco** [comment]
> TagHelpers that might be used in the email templates? 

Which templates that you refer to?

**Comment by hishamco** [comment]
I will remove the last commit that suggested by Jasmin, seems I didn't see there's a use of resources until Dean shows the `at` in script tag

**Comment by hishamco** [comment]
Again @Skrypt which email templates that you refer to?

**Comment by Skrypt** [comment]
The ones from Workflow Tasks.

**Comment by jptissot** [comment]
I think this should be closed @hishamco

**Comment by hishamco** [comment]
> As I said you can remove the dependency in the manfest (probably). but that is the only change we can accept

@jptissot I will change this PR to remove the dependency from the manifest

**Comment by hishamco** [comment]
> @jptissot I will change this PR to remove the dependency from the manifest

Seems I already remove it ;)

@deanmarcussen can we approve this?

**Comment by hishamco** [comment]
@Skrypt can we merge this, it takes so long ;)

File: src/OrchardCore.Modules/OrchardCore.Email/Manifest.cs [hunk 1]
```diff
@@ -6,6 +6,5 @@
     Website = ManifestConstants.OrchardCoreWebsite,
     Version = ManifestConstants.OrchardCoreVersion,
     Description = "Provides email settings configuration and a default email service based on SMTP.",
-    Dependencies = new[] { "OrchardCore.Resources" },
```
Comment (Piedone) [inline src/OrchardCore.Modules/OrchardCore.Email/Manifest.cs:9]: The module [uses CodeMirror](https://github.com/OrchardCMS/OrchardCore/blob/ae2f9525e29f533b1a993d7279c9a45d06ac5198/src/OrchardCore.Modules/OrchardCore.Email/Views/Items/EmailTask.Fields.Edit.cshtml#L67) from the Resources module, so the dependency is warranted, even if in most scenarios, due to the Resources module being enabled anyway, this wouldn't cause an issue.

**Comment by Piedone** [comment]
There's nothing to be merged here it seems. Please reopen if necessary.

**Pull request closed without merging**